hpotter-server/hpotter/test/test_https.py

```diff
@@ -0,0 +1,21 @@
+import ssl, socket, requests
+
+hostname = '127.0.0.1'
+data = b'test'
+url = 'http://127.0.0.1'
+
+context = ssl.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
+context.load_verify_locations('/Users/mattmcmahon/Desktop/cert.pem')
```

Comment:
You have a hard coded path and this is not going to work on anyones elses system. Also we don't have this cert so the test will always fail for everyone other then you

Reply:
haha whoops, I'll move the cert into test/